tests/stable_api_test.py

```diff
@@ -12,6 +12,33 @@
 from tests.utils import Constructor
 from tests.utils import assert_equal_data
 
+DTYPES = {
```

Comment:
just out of interest, why add this?

Reply:
hey Marco, I wasn't updating, I was just trying to resolve some conflicts and update my fork/local repo. I was working on the dtypes file and also had to adjust the tests and then you did some changes to the test file (among those was adding this DTYPES = {....}-thing, I think you did it during one of the live streams.)

Reply:
thanks for resolving merge conflicts then 🙌 

i think it might not be necessary any more to have `DTYPES` here then, could you try removing it from this file?

Reply:
Yes, I think I first merged your changes and then committed them back, in the process of updating my local repo. I can see now that you refactored this test 2 weeks ago or so. Yes, I'll try to get it right!